respx/api.py

```diff
@@ -1,12 +1,21 @@
 from typing import Callable, Optional, Pattern, Union, overload
 
 from .mocks import MockTransport
-from .models import CallList, ContentDataTypes, DefaultType, HeaderTypes, RequestPattern
+from .models import (
+    CallList,
+    ContentDataTypes,
+    DefaultType,
+    HeaderTypes,
+    RequestPattern,
+    _deprecate_object,
+)
 
 mock = MockTransport(assert_all_called=False)
 
 aliases = mock.aliases
-stats = mock.stats
+stats = _deprecate_object(
```

Comment:
I faced a problem here. In the previous version, I had here `stats = mock.stats`. It calls `stats` property and throws a warning. So, for getting the warning you only need to import API :) I tried to find any solution to fix it and finally came up with this crazy solution with metaprogramming. I don't like it, but it works as expected.

We can merge it and delete `respx.stats` at the next release or we can just remove it now, w/o deprecation warning. 

Or, mb, you can come up with a cleaner solution